homeassistant/components/spc/__init__.py

```diff
@@ -16,61 +17,80 @@
 CONF_API_URL = "api_url"
 
 DOMAIN = "spc"
-DATA_API = "spc_api"
 
 SIGNAL_UPDATE_ALARM = "spc_update_alarm_{}"
 SIGNAL_UPDATE_SENSOR = "spc_update_sensor_{}"
+PLATFORMS = ["alarm_control_panel", "binary_sensor"]
 
 CONFIG_SCHEMA = vol.Schema(
     {
```

Comment:
This whole thing can be simplified:

```python
CONFIG_SCHEMA = cv.deprecated(DOMAIN, invalidation_version="0.119")
```

Reply:
If you want the import to work you can't change this, but add a target release to end support with

Reply:
Thanks. invalidation_version added.